### PR TITLE
corrected vagrant installation for BeLL-App installation guide

### LINKED ISSUE #1506: fix vagrant install for bash script
screenshot from 2018-01-16 15-05-49

```
sudo apt install vagrant
``` 
The `vagrant` package has been deprecated, and has to be installed using a different method.

I found a way to do it using the command line, using [this source](https://tutorialforlinux.com/2017/10/09/how-to-install-vagrant-ubuntu-17-10-artful-linux-easy-guide/).

I'll just add these steps to the script located on the BeLL-App installation page.

### PR DESCRIPTION
**Pull request by anagabriel42**
closes #1506

### CODE REVIEW

File: pages/techgenius/tg-installation.md [hunk 1]
```diff
@@ -62,7 +62,10 @@ If it does not work, try this:
 ```bash
     sudo apt-get install git
     sudo apt-get install virtualbox
-    sudo apt-get install vagrant
+    curl -o vagrant.deb https://releases.hashicorp.com/vagrant/2.0.1/vagrant_2.0.1_x86_64.deb?_ga=2.145355697.407736321.1516146560-799695961.1515792702
```
Comment (i5o) [inline pages/techgenius/tg-installation.md]: I think that you should remove the ?_ga in this case, the links works without it anyway

File: pages/techgenius/tg-installation.md [hunk 2]
```diff
@@ -62,7 +62,10 @@ If it does not work, try this:
...
+    sudo apt-get install gdebi*
```
Comment (i5o) [inline pages/techgenius/tg-installation.md]: you also don't need this, you could install the package using "dpkg"

File: pages/techgenius/tg-installation.md [hunk 3]
```diff
@@ -62,7 +62,10 @@ If it does not work, try this:
...
+    sudo gdebi ./vagrant*.deb
```
Comment (i5o) [inline pages/techgenius/tg-installation.md]: sudo dpkg -i vagrant.deb

File: pages/uploads/bashScripts/instructions.sh [hunk 4]
```diff
@@ -2,7 +2,10 @@
 
 sudo apt-get install git  #Install git if it is not already present
 sudo apt-get install virtualbox #Install virtualbox if it is not already present
-sudo apt-get install vagrant #Install vagrant if it is not already present
+curl -o vagrant.deb https://releases.hashicorp.com/vagrant/2.0.1/vagrant_2.0.1_x86_64.deb?_ga=2.145355697.407736321.1516146560-799695961.1515792702
```
Comment (i5o) [inline pages/uploads/bashScripts/instructions.sh]: same comments I wrote before apply for this case

File: pages/techgenius/tg-installation.md (hunk 3 above)
Comment (i5o) [reply pages/techgenius/tg-installation.md]: you may want to run

sudo apt-get install -fy

to fix dependencies if any is missing

File: pages/techgenius/tg-installation.md (hunk 1 above)
Comment (anagabriel42) [reply pages/techgenius/tg-installation.md]: sounds good!

File: pages/techgenius/tg-installation.md (hunk 3 above)
Comment (anagabriel42) [reply pages/techgenius/tg-installation.md]: so, I could just leave it as is then?

Comment (anagabriel42) [reply pages/techgenius/tg-installation.md]: sounds good! :)

**Comment by anagabriel42** [comment]
@i5o I updated the files!

File: pages/uploads/bashScripts/instructions.sh [hunk 5]
```diff
@@ -2,7 +2,10 @@
 
 sudo apt-get install git  #Install git if it is not already present
 sudo apt-get install virtualbox #Install virtualbox if it is not already present
-sudo apt-get install vagrant #Install vagrant if it is not already present
+curl -o vagrant.deb https://releases.hashicorp.com/vagrant/2.0.1/vagrant_2.0.1_x86_64.deb
+sudo apt-get install -fy
+sudo dpkg -i vagrant.deb
```
Comment (i5o) [inline pages/uploads/bashScripts/instructions.sh]: Same comments as before here

File: pages/uploads/bashScripts/instructions.sh [hunk 6]
```diff
@@ -2,7 +2,10 @@
...
+vagrant -h #Install vagrant if it is not already present
```
Comment (i5o) [inline pages/uploads/bashScripts/instructions.sh]: vagrant -h

shows the help of vagrant; maybe we should rewrite the "#install vagrant.." to "# Check if vagrant is present"

File: pages/techgenius/tg-installation.md [hunk 7]
```diff
@@ -62,7 +62,10 @@ If it does not work, try this:
 ```bash
     sudo apt-get install git
     sudo apt-get install virtualbox
-    sudo apt-get install vagrant
+    curl -o vagrant.deb https://releases.hashicorp.com/vagrant/2.0.1/vagrant_2.0.1_x86_64.deb
+    sudo apt-get install -fy
```
[the remark this answers is not in the document]
Comment (anagabriel42) [reply pages/techgenius/tg-installation.md]: ahhh gotcha!

**Comment by anagabriel42** [comment]
@i5o updated the files again! :)

**Comment by anagabriel42** [comment]
thank you, everyone!

**Pull request merged**